<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AIvsAI</class>
 <widget class="QWidget" name="AIvsAI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="mapButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>载入地图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AiButton1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>载入AI1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AiButton2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>载入AI2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>563</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>610</y>
     <width>141</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>返回上级</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AiCombo1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>273</y>
     <width>250</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="AiCombo2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>353</y>
     <width>250</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mapCombo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>433</y>
     <width>250</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="roundLCD">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 回合数：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
